src/pages/Login/LoginPage.scss

```diff
@@ -1,59 +1,93 @@
 
-.login-page {
+.login-page{
     display: flex;
     flex-direction: row;
-    height: 100vh;
+    height: 100%;
+    width: 100%;
 
-    .left-column {
-
-        .landing-image {
-            width: 27vw;
-            height: 100%;
+    .left-container{
+        display: flex;
+        flex-direction: column;
+        height: 100vh;
+        width: 20vw;
+        
+        .andela-logo{
+            margin: 20% 0% 0% 10%;
+            width: 7.7vw;
+            height: 4.5vh;
         }
-    }
 
-    .right-column {
-        flex: 1;
+        .welcome-text{
+            margin-top: 40vh;
+            margin-left: 2vw;
+            width: 70%;
 
-        .login-container {
-            display: flex;
-            flex-direction: column;
-            align-items: center;
-            height: 100vh;
-            justify-content: center;
-            margin: auto !important;
 
-            .title {
-                margin-bottom: 0.3rem;
-                font-weight: 200;
-                font-size: 1.25rem;
-            }
+            p{
+                font-family: 'DIN Pro Extra Light';
+                font-size: 25px;
 
-            .instructions {
-                margin: 0.2rem;
+                .wire{
+                    font-family: 'DIN Pro Light';
+                    font-style: italic;
+                }
+                
             }
+        }
+        .button{
+            
+            .google-logo{
+                display: inline-block;
+                width: 3vw;
+                height: 60%;
+                padding: 0%;
+                margin-left: 0%;
 
-            .google-logo {
+            }
+            .label{
                 display: inline-block;
-                position: relative;
-                right: 0.5rem;
-                width: 3.5rem;
-                height: 80%;
-                box-sizing: border-box;
             }
+        }
 
-            .button {
-                margin: 1rem;
-                border: 1px solid map-get($grey, 100);
-                font-size: small;
-                padding: 0.1rem 0.1rem;
-            }
+     }
+        
+
+
+    .right-container{
+        display: flex;
+        flex-direction: column;
+        height: 100vh;
+        width: 80vw;
+
+
+        .landing-image{
+            height: 100vh;
+            width: 80vw;
+        }
+
+        .right-text{
+            position: absolute;
+            color: #fbaf31;
+            font-size: 40px;
+            font-weight: 500;
+            margin-left: 45vw;
+            margin-top: 23vh;
 
-            .landing-logo {
-                height: 5rem;
-                margin-top: 1rem;
-            }
         }
+            .underline{
+                position: absolute;
+                background-color: #fdb237;
+                width: 8vw;
+                height: 0.5vh;
+                border-radius: 5px;
+                left: 68vw;
+                top: 39vh;
+
```

Comment:
Remove the unnecessary spaces

Reply:
Also, Indent your code your scss format is incorrect  lets follow the style guidlines here is the link https://sass-guidelin.es/#syntax--formatting

Reply:
Cool , I shall sort that out